01_01_Basic_Rules.txt

```diff
@@ -1038,7 +1028,8 @@ of 7-10 would be successful.
 Gun Jams:
 	If at any time that you are firing a gun and you roll a weapon accuracy of 1,
 roll a second d10. If that second d10 roll is a 1 or 2, the weapon Jams, 
-necessitating a weapon skill check, DC 60. 
+necessitating a Weapon [X] skill check, DC 60. Your GM may allow you to add skill in
+Repair to this check.

```

Comment:
Balance concern: letting repair help clear jams in the thick of combat devalues weapon skills.

Reply:
I just thought it seemed logical story-wise. Not attached to the idea.

Reply:
I grant that it may be misplaced, but I have faith in players with high repair and no weapon skill to implore their DM for leniency whether prompted or not, because problem solving. Just hesitant for the rules to officially endorse it. 
However this shouldn't block the merge by itself. It can be revisited later.

Reply:
Nah, I'll yank it :)